docker/dev/angularjs/Dockerfile

```diff
@@ -1,10 +1,8 @@
-FROM node:9.4.0
-MAINTAINER CloudCV Team
+FROM node:12.2.0
 
-RUN apt-get update -qq && apt-get install -y build-essential git curl libfontconfig
+RUN apt-get update
+RUN apt-get install -y build-essential git curl libfontconfig
 RUN apt-get install nodejs-legacy -y
-RUN apt-get install npm -y
-RUN npm cache verify
 RUN npm install -g @angular/cli@1.6.5 --unsafe
```

Comment:
We can use latest version, no need to specify version. Actually latest version can work for previous angular version project as well.

Reply:
Good catch, Thanks!